.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Describe your change
+
+<!--- Describe your change here-->
+
+### Related Issue
+<!--- This project prefers to accept pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- Please [link to the issue here](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue) by writing "resolves #123456" here: -->
+
+## Contributor License Agreement
+
+<!--- All contributions to FDC3 must be made under an active contributor license agreement and the [Community Specification License](https://github.com/finos/FDC3/blob/main/LICENSE.md). This will be checked by the EasyCLA tool (https://easycla.lfx.linuxfoundation.org/) that runs automatically on every PR. If you've not contributed to FDC3 before, look for a comment on your PR shortly after it is raised and follow the instructions to establish a CLA or have it acknowledged by the EasyCLA tool. -->
+
+- [ ] I acknowledge that a contributor license agreement is required and that I have one in place or will seek to put one in place ASAP.
+
+## Review Checklist
+
+<!--- Checklist to be completed by reviewers, and pre-checked by the authors of a PR -->
+
+- [ ] Is a CHANGELOG.md entry included?
```

Comment:
Similar to this, do we want to explicitly call out an issue being included in the "Related Issue" section?

Reply:
```suggestion
- [ ] If a change was made to the FDC3 Standard, was an issue linked above?
- [ ] Is a CHANGELOG.md entry included?
```